.ci/perf_tests/dockerfiles/perf_tests.Dockerfile

```diff
@@ -15,7 +15,7 @@ ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
     FUNCTIONS_WORKER_SHARED_MEMORY_DATA_TRANSFER_ENABLED=1
 
 RUN apt-get --quiet update && \
-    apt-get install --quiet -y git && \
+    apt-get install --quiet -y git procps && \
```

Comment:
Can we add a comment somewhere what does the packages do such that context is preserved.